pkgs/development/tools/comby/default.nix

```diff
@@ -0,0 +1,66 @@
+{ ocamlPackages, fetchFromGitHub, lib, zlib, pkg-config }:
+let
+  mkCombyPackage = { pname, extraBuildInputs ? [ ], extraNativeInputs ? [ ] }:
+    ocamlPackages.buildDunePackage rec {
+      inherit pname;
+      version = "1.5.1";
+      useDune2 = true;
+      minimumOcamlVersion = "4.08.1";
+
+      src = fetchFromGitHub {
+        owner = "comby-tools";
+        repo = "comby";
+        rev = version;
+        sha256 = "1ipfrr6n1jyyryhm9zpn8wwgzfac1zgbjdjzrm00qcwc17r8x2hf";
+      };
+
+      nativeBuildInputs = [
+        ocamlPackages.ppx_deriving
+        ocamlPackages.ppx_deriving_yojson
+        ocamlPackages.ppx_sexp_conv
+        ocamlPackages.ppx_sexp_message
+      ] ++ extraNativeInputs;
+
+      buildInputs = [
+        ocamlPackages.core
+        ocamlPackages.ocaml_pcre
+        ocamlPackages.mparser
+        ocamlPackages.mparser-pcre
+      ] ++ extraBuildInputs;
+
+      meta = {
+        description = "Tool for searching changing code structure";
+        license = lib.licenses.asl20;
+        homepage = "https://comby.dev";
+      };
+    };
+
+  combyKernel = mkCombyPackage {
+    pname = "comby-kernel";
+    extraBuildInputs = [ ocamlPackages.re ocamlPackages.angstrom ];
+    extraNativeInputs = [ ocamlPackages.ppx_here ];
```

Comment:
neither `ppx_here` nor `re` are listed in the opam file, are they really necessary (sometimes there are discrepancies though)?

Reply:
No they’re not, I removed them.